game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -15,8 +15,7 @@ object Cell {
   final val MinMass = 1 // TODO determine this once we have visualization/etc.
 }
 
-class Cell(id: Int, startPosition: Vector2 = new Vector2(0f, 0f)) {
-  var position = startPosition
+class Cell(id: Int, grid: Grid, startPosition: Vector2 = new Vector2(0f, 0f)) {
```

Comment:
Do you dislike the current solution?

Hmmm, you prefer moving the responsability out of the cell? Do you think it's preferable?

I'm unsure about both, because it would still be possible to have cell with a bad state. Is it acceptable?